functionnal_tests/cypress/integration/app_thread/translation.js

```diff
@@ -0,0 +1,26 @@
+import { PAGES } from '../../support/urls_commands'
+
+describe('navigate :: workspace > create_new > thread', function () {
+  before(() => {
+    cy.resetDB()
+    cy.setupBaseDB()
+    cy.loginAs('users')
+    cy.fixture('baseWorkspace').as('workspace').then(workspace => {
+      cy.visitPage({
+        pageName: PAGES.CONTENTS,
+        getters: { type: 'thread' },
+        params: { workspaceId: workspace.workspace_id }
+      })
+    })
+  })
+
+  it('should have translations', () => {
+    cy.get('.workspace__header__title').contains('List of threads')
```

Comment:
This is testing the workspace, not the app thread. Use the same logic you did in `functionnal_tests/cypress/integration/app_file/translation.js` .

Reply:
fixed, I overlooked this one, thanks